app/liquid_tags/github_tag/github_readme_tag.rb

```diff
@@ -4,38 +4,35 @@ class GithubTag
   class GithubReadmeTag
     PARTIAL = "liquids/github_readme".freeze
 
+    attr_reader :client, :content, :options, :readme_html
+
     def initialize(link)
-      @link = parse_link(link)
+      parsed_link = parse_link(link)
       @options = parse_options(link)
-      @content = get_content(@link)
+      @client = Octokit::Client.new(access_token: token)
+      @content = client.repository(parsed_link)
```

Comment:
After splitting responsibilities and relying less on different instance variables being set, the `get_content` method was no longer necessary, so I directly call the client here.